src/transactions/feed/TransactionFeed.tsx

```diff
@@ -103,11 +103,23 @@ function TransactionFeed() {
   function renderItem({ item: tx }: { item: FeedTokenTransaction; index: number }) {
     switch (tx.__typename) {
       case 'TokenExchangeV2':
-        return <SwapFeedItem key={tx.transactionHash} exchange={tx} />
+        return (
+          <View testID="TransactionListItem">
+            <SwapFeedItem key={tx.transactionHash} exchange={tx} />
+          </View>
+        )
       case 'TokenTransferV2':
-        return <TransferFeedItem key={tx.transactionHash} transfer={tx} />
+        return (
+          <View testID="TransactionListItem">
+            <TransferFeedItem key={tx.transactionHash} transfer={tx} />
+          </View>
+        )
       case 'NftTransferV2':
-        return <NftFeedItem key={tx.transactionHash} transaction={tx} />
+        return (
+          <View testID="TransactionListItem">
+            <NftFeedItem key={tx.transactionHash} transaction={tx} />
+          </View>
```

Comment:
I'd prefer we pass a `testID` and modify these components instead of wrapping them with an unnecessary view.

It's a micro optimization, but if it's not strictly needed, let's avoid the extra view.

Reply:
Removed in https://github.com/valora-inc/wallet/pull/3192/commits/d893332e71e3946ec1209ad84fbec663686d6373!